resources/api-gateway-migrator/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
@colunira could we install it only during kyma upgrade? it doesn't make sense to install it during installation, there will be nothing to migrate anyway?

Reply:
Helm doesn't see if we install or upgrade the whole Kyma, just charts, so it is not possible.